### PR TITLE
Update release-notes for release-v1.8.0

### PR DESCRIPTION
**Pull request by csmbot**
Update release-notes to include features and bug fixes

### CODE REVIEW

File: content/docs/csidriver/release/powerflex.md [hunk 1]
```diff
@@ -6,19 +6,17 @@ description: Release notes for PowerFlex CSI driver
...
 - [#876 - [FEATURE]: CSI 1.5 spec support -StorageCapacityTracking](https://github.com/dell/csm/issues/876)
-- [#926 - [FEATURE]: Set up golangci-lint for all CSM repositories](https://github.com/dell/csm/issues/926)
 - [#947 - [FEATURE]: K8S 1.28 support in CSM 1.8](https://github.com/dell/csm/issues/947)
-- [#878 - [FEATURE]: CSI 1.5 spec support : Implement Volume Limits](https://github.com/dell/csm/issues/878)
-- [#885 - [FEATURE]: SDC 3.6.1 support](https://github.com/dell/csm/issues/885)
+- [#878 - [FEATURE]: CSI 1.5 spec support: Implement Volume Limits](https://github.com/dell/csm/issues/878)
 
 ### Fixed Issues
 
-- [#916 - [BUG]: Remove refs to deprecated io/ioutil](https://github.com/dell/csm/issues/916)
```
Comment (VamsiSiddu-7) [inline content/docs/csidriver/release/powerflex.md:21]: This should not be removed. @adarsh-dell  the issue has everything you mentioned in the channel.

File: content/docs/csidriver/release/powerflex.md [hunk 2]
```diff
@@ -6,19 +6,17 @@ description: Release notes for PowerFlex CSI driver
 ## Release Notes - CSI PowerFlex v2.8.0
 
 
+
 ### New Features/Changes
 
 - [#724 - [FEATURE]: CSM support for Openshift 4.13](https://github.com/dell/csm/issues/724)
 - [#763 - [FEATURE]: CSI-PowerFlex 4.0 NFS support](https://github.com/dell/csm/issues/763)
 - [#876 - [FEATURE]: CSI 1.5 spec support -StorageCapacityTracking](https://github.com/dell/csm/issues/876)
-- [#926 - [FEATURE]: Set up golangci-lint for all CSM repositories](https://github.com/dell/csm/issues/926)
```
Comment (VamsiSiddu-7) [inline content/docs/csidriver/release/powerflex.md:14]: This also should not get removed.

File: content/docs/csidriver/release/powerflex.md [hunk 3]
```diff
@@ -6,19 +6,17 @@ description: Release notes for PowerFlex CSI driver
...
 - [#947 - [FEATURE]: K8S 1.28 support in CSM 1.8](https://github.com/dell/csm/issues/947)
-- [#878 - [FEATURE]: CSI 1.5 spec support : Implement Volume Limits](https://github.com/dell/csm/issues/878)
-- [#885 - [FEATURE]: SDC 3.6.1 support](https://github.com/dell/csm/issues/885)
```
Comment (VamsiSiddu-7) [inline content/docs/csidriver/release/powerflex.md:17]: This also should not get removed.

**Review by VamsiSiddu-7 — CHANGES_REQUESTED** [review]

File: content/docs/csidriver/release/powerflex.md (hunk 2 above)
Comment (adarsh-dell) [reply content/docs/csidriver/release/powerflex.md:14]: @VamsiSiddu-7 Milestone is not added for this issue, if you want this to be added here, please add milestone from your end.

File: content/docs/csidriver/release/powerflex.md (hunk 3 above)
Comment (adarsh-dell) [reply content/docs/csidriver/release/powerflex.md:17]: @VamsiSiddu-7 Similarly, the milestone hasn't been added here either. I kindly request that each team takes the initiative to add it from their end. I'm unsure if this is considered part of the CSM 1.8 deliverable, so your assistance in this case is greatly appreciated.

File: content/docs/csidriver/release/powerflex.md (hunk 1 above)
Comment (adarsh-dell) [reply content/docs/csidriver/release/powerflex.md:21]: Nice point, let me look into this.

File: content/docs/csidriver/release/powerflex.md (hunk 2 above)
Comment (VamsiSiddu-7) [reply content/docs/csidriver/release/powerflex.md:14]: added the milestone

File: content/docs/csidriver/release/powerflex.md (hunk 1 above)
Comment (adarsh-dell) [reply content/docs/csidriver/release/powerflex.md:21]: @VamsiSiddu-7 Milestone should be v1.8.0 not CSM 1.8.
Please work with the respective owners and get it rectified.

**Pull request closed without merging**